internal/query/query.go

```diff
@@ -79,24 +78,25 @@ func StartQueries(ctx context.Context, es *eventstore.Eventstore, projections pr
 	return repo, nil
 }
 
-func (r *Queries) IAMByID(ctx context.Context, id string) (_ *iam_model.IAM, err error) {
-	readModel, err := r.iamByID(ctx, id)
-	if err != nil {
-		return nil, err
-	}
-
-	return readModelToIAM(readModel), nil
-}
-
-func (r *Queries) iamByID(ctx context.Context, id string) (_ *ReadModel, err error) {
-	ctx, span := tracing.NewSpan(ctx)
-	defer func() { span.EndWithError(err) }()
-
-	readModel := NewReadModel(id)
-	err = r.eventstore.FilterToQueryReducer(ctx, readModel)
-	if err != nil {
-		return nil, err
-	}
-
-	return readModel, nil
-}
+//
+//func (r *Queries) IAMByID(ctx context.Context, id string) (_ *iam_model.IAM, err error) {
+//	readModel, err := r.iamByID(ctx, id)
+//	if err != nil {
+//		return nil, err
+//	}
+//
+//	return readModelToIAM(readModel), nil
+//}
+//
+//func (r *Queries) iamByID(ctx context.Context, id string) (_ *ReadModel, err error) {
+//	ctx, span := tracing.NewSpan(ctx)
+//	defer func() { span.EndWithError(err) }()
+//
+//	readModel := NewReadModel(id)
+//	err = r.eventstore.FilterToQueryReducer(ctx, readModel)
+//	if err != nil {
+//		return nil, err
+//	}
+//
+//	return readModel, nil
+//}
```

Comment:
please remove this and all old ReadModels (iam_model.go, ...) in the query package

Reply:
done